src/components/component/component-identity.tsx

```diff
@@ -59,12 +58,27 @@ const AzureIdentityLink = ({
           Azure Federated Credentials Configuration
         </Popover.Header>
         <Popover.Content>
-          <AzureIdentity
-            oidcIssuerUrl={configVariables.CLUSTER_OIDC_ISSUER_URL}
-            clientId={clientId}
-            namespace={namespace}
-            serviceAccountName={serviceAccountName}
-          />
+          <div className="grid grid--gap-medium">
+            <AzureIdentity
+              oidcIssuerUrl={configVariables.CLUSTER_OIDC_ISSUER_URL}
+              clientId={clientId}
+              namespace={namespace}
+              serviceAccountName={serviceAccountName}
+              azureKeyVaults={azureKeyVaults}
+            />
+          </div>
+          {azureKeyVaults?.length > 0 && (
+            <div className="grid grid--gap-medium">
+              <Typography className="whitespace-nowrap" variant="h6" as="span">
+                Azure Key Vaults using Azure identity
+              </Typography>
+              <List variant="bullet">
+                {azureKeyVaults.map((name: string) => (
+                  <List.Item key={name}>{name}</List.Item>
+                ))}
+              </List>
+            </div>
+          )}
```

Comment:
```suggestion
          <div className="grid grid--gap-medium">
            <AzureIdentity
              oidcIssuerUrl={configVariables.CLUSTER_OIDC_ISSUER_URL}
              clientId={clientId}
              namespace={namespace}
              serviceAccountName={serviceAccountName}
            />
            {azureKeyVaults?.length > 0 && (
              <div className="grid grid--gap-small">
                <Typography
                  className="whitespace-nowrap"
                  variant="h6"
                  as="span"
                >
                  Azure Key Vaults using Azure identity
                </Typography>
                <List variant="bullet">
                  {azureKeyVaults.map((name: string) => (
                    <List.Item key={name}>{name}</List.Item>
                  ))}
                </List>
              </div>
            )}
          </div>
```
Suggest we wrap all elements within Popover.Content inside a grid with medium gap, and then Azure KeyVault info inside a grid with small gap.

Also removed unused azureKeyVaults prop for AzureIdentity component.